<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCtrlPage</class>
 <widget class="QWidget" name="QCtrlPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <strikeout>true</strikeout>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QProgressBar {
     border: 2px solid grey;
     border-radius: 5px;
     text-align: center;
 }

QProgressBar::chunk {
     background-color: #05B8CC;
     width: 1px;
}

QCheckBox:!hover{
	color:white;
	border-radius:10px;
	border:1px solid rgb(170, 170, 127);
	background-color: rgb(255, 170, 127);
	font: 12pt &quot;楷体&quot;;
}

QCheckBox:hover{
	color:white;
	font: 12pt &quot;楷体&quot;;
	border-radius:10px;
	border:1px solid rgb(170, 170, 127);
	background-color: rgb(170, 240, 0);
}


/* === QPushButton === */
QPushButton {
    border: 1px solid #555;
    padding: 4px;
    min-width: 65px;
    min-height: 12px;
	border-radius:5px;
	background-color:#70B1B3;
}

QPushButton:hover {
    background-color: #999;
}

QPushButton:pressed {
    background-color: #333;
    border-color: #555;
    color: #AAA;
}

QPushButton:disabled {
    color: #333333;
	background-color:#DBDBDB;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>480</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 176, 181);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>251</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="title">
    <string>功能控制</string>
   </property>
   <widget class="QPushButton" name="bt_allOpen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>全亮</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_allClose">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>全灭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_closeAuto">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>关闭光感调节</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_openAuto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>开启光感调节</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_closeGesture">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>关闭手势调节</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_openGesture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>开启手势调节</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>280</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>146</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>bt_allOpen</zorder>
   <zorder>bt_allClose</zorder>
   <zorder>bt_closeAuto</zorder>
   <zorder>bt_openAuto</zorder>
   <zorder>bt_closeGesture</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>bt_openGesture</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>491</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="title">
    <string>灯光控制</string>
   </property>
   <widget class="QPushButton" name="bt_red">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>红</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icon/red.png</normaloff>:/icon/red.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_yellow">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>黄</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icon/yellow.png</normaloff>:/icon/yellow.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_blue">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>蓝</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icon/blue.png</normaloff>:/icon/blue.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_green">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>绿</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icon/green.png</normaloff>:/icon/green.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_cyan">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>浅蓝</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icon/cyan.png</normaloff>:/icon/cyan.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_white">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>白</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icon/white.png</normaloff>:/icon/white.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_grey">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>true</strikeout>
     </font>
    </property>
    <property name="text">
     <string>灰</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icon/grey.png</normaloff>:/icon/grey.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_purple">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>紫</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icon/purple.png</normaloff>:/icon/purple.png</iconset>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizon {
    background: red;
    position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */
    left: 4px; 
	right: 4px;
}

QSlider::handle:horizon {
    height: 10px;
    background: cyan;
    margin: 0 -8px; /* expand outside the groove */
}

QSlider::add-page:horizon {
    background: white;
}

QSlider::sub-page:horizon {
    background: pink;
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_state_led1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icon/grey.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_value2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_state_led2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icon/grey.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LED1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LED2</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_led1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_led2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <zorder>bt_red</zorder>
   <zorder>bt_yellow</zorder>
   <zorder>bt_blue</zorder>
   <zorder>bt_green</zorder>
   <zorder>bt_cyan</zorder>
   <zorder>bt_white</zorder>
   <zorder>bt_grey</zorder>
   <zorder>bt_purple</zorder>
   <zorder>lb_state_led1</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>lb_value2</zorder>
   <zorder>lb_state_led2</zorder>
   <zorder>checkBox_1</zorder>
   <zorder>checkBox_2</zorder>
   <zorder>progressBar_led1</zorder>
   <zorder>progressBar_led2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
